src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityTests.java

```diff
@@ -79,6 +79,7 @@ public void testAllVersionsTested() throws Exception {
                 Version v = (Version)field.get(Version.class);
                 if (v.snapshot()) continue;
                 if (v.onOrBefore(Version.V_0_20_6)) continue;
+                if (v.equals(Version.CURRENT)) continue;
```

Comment:
I think a function is overkill, having the logic in the loop is nice and direct. I added comments.